prov/netdir/src/netdir_init.c

```diff
@@ -118,7 +120,7 @@ static int ofi_nd_adapter_cb(const ND2_ADAPTER_INFO *adapter, const char *name)
 	info->fabric_attr->prov_version = FI_VERSION(OFI_ND_MAJOR_VERSION, OFI_ND_MINOR_VERSION);
 
 	info->caps = OFI_ND_CAPS;
-	info->addr_format = FI_SOCKADDR;
+	info->addr_format = FI_SOCKADDR | FI_SOCKADDR_IN | FI_SOCKADDR_IN6;
```

Comment:
fi_sockaddr covers sockaddr_in and sockaddr_in6.  Since sockaddr isn't actually a real address format, but a placement for any sockaddr_xx type, this is fine.